### PR TITLE
GPG plugin: use m/0/0 as ecc gpg key

### PR DESCRIPTION
**Pull request by asfin**
Hi.
Got request from user to add this functionality.
Plugin adds modal window with popular gpg operations - sign/verify, encrypt/decrypt, view key.
Tested on fresh ubuntu, additionally requires `libgpgme-dev` and `swig` to build. (Q1:  How i should include this deps into project manifests?)
Tried to build this for Windows, but can't finish `pip install gpg`, because system gpg cannot be found. (Q2: Is there any way to fix it? Temporary left some changes with this in diff).
Also i'll check OSX build before merge.

### CODE REVIEW

**Comment by Hasimir** [comment]
The best way to resolve this would be for the Electrum developers to contact the GnuPG Project Python bindings maintainer directly in order to implement this correctly.

So it's a good thing that GnuPG coder just found this feature request and would also like to see it implemented.  ;)

**Comment by romanz** [comment]
I can also suggest using https://github.com/romanz/trezor-agent for that :)

File: contrib/build-wine/deterministic.spec [hunk 1]
```diff
@@ -1,139 +1,141 @@
...
-    a.binaries,
-    a.zipfiles,
-    a.datas,
-    strip=None,
-    upx=True,
-    debug=False,
-    icon=home+'icons/electrum.ico',
-    console=False,
-    name=os.path.join('dist', 'electrum'))
+# -*- mode: python -*-
```
[the remark this answers is not in the document]
Comment (asfin) [reply contrib/build-wine/deterministic.spec]: Looks like this file has windows endings, and my merge tool changed it to unix. Reverted merge, changed only one line.

File: contrib/build-wine/prepare-wine.sh [hunk 2]
```diff
@@ -13,6 +13,14 @@ LIBUSB_FILENAME=libusb-1.0.22.7z
 LIBUSB_URL=https://prdownloads.sourceforge.net/project/libusb/libusb-1.0/libusb-1.0.22/$LIBUSB_FILENAME?download
 LIBUSB_SHA256=671f1a420757b4480e7fadc8313d6fb3cbb75ca00934c417c1efa6e77fb8779b
 
+GPG_FILENAME=gpg4win-3.1.0.exe
+GPG_URL=https://files.gpg4win.org/$GPG_FILENAME
+GPG_SHA256=a416aca375a80eada7edc69efdf3757988310a2f1723cd6190d9e3113d787301
+
+GPG_FILENAME=gnupg-w32-2.2.7_20180502.exe
+GPG_URL=https://www.gnupg.org/ftp/gcrypt/binary/$GPG_FILENAME
+GPG_SHA256=2d7ffedb1737e4dbed7f89377c3273b5cc27ee0fd9c70a417bd602203fc419ee
```
[the remark this answers is not in the document]
Comment (asfin) [reply contrib/build-wine/prepare-wine.sh]: Debugging wine build. Fix this lines as i'll get working windows installer.

**Comment by asfin** [comment]
Got new trouble, when i build windows executable(with docker) even on upstream/master, plugins window shows nothing. It's known bug?

**Comment by SomberNight** [comment]
Ahhhhh. Thanks for saying!

**Comment by SomberNight** [comment]
Try now with https://github.com/spesmilo/electrum/commit/e9f18883211cf823815580df9dfa8ac254ab43f7

**Comment by asfin** [comment]
Thanks, much better. Windows version works fine. I need to check Mac version, and after this i'm ready to merge.

**Comment by accumulator** [comment]
A plugin like this is more suitable as an external plugin. See also #9235

**Pull request closed without merging**